python/paddle/fluid/layer_helper.py

```diff
@@ -397,7 +399,7 @@ def append_activation(self, input_var):
         act = self.kwargs.get('act', None)
         if act is None:
             return input_var
-        if isinstance(act, basestring):
+        if isinstance(act, six.string_types):
```

Comment:
doesn't cover bytes in py3?

Reply:
never meet ```bytes``` here if we decoded the args' name above

Reply:
else raise Error?